dwave/preprocessing/composites/fix_variables.py

```diff
@@ -45,6 +45,9 @@ class FixVariablesComposite(ComposedSampler):
             variables the algorithm will fix. For details, see 
             :func:`~dwave.preprocessing.lower_bounds.roof_duality`.
 
+    .. versionadded:: 0.6.8
+        Support for context manager protocol and :meth:`.close` method.
+
     Examples:
        This example uses the :class:`.FixVariablesComposite` to instantiate a
        composed sampler that submits a simple Ising problem to a sampler.
```

Comment:
line 61: `sampler = FixVariablesComposite(ExactSolver())` outside of a context manager and not closed